libdeno/BUILD.gn

```diff
@@ -5,9 +5,6 @@ import("//third_party/v8/gni/v8.gni")
 config("deno_config") {
   include_dirs = [ "//third_party/v8" ]  # This allows us to v8/src/base/ libraries.
   configs = [ "//third_party/v8:external_config" ]
-  if (is_debug) {
-    defines = [ "DEBUG" ]
```

Comment:
IIRC this is for something related to libv8_base's `CHECK` and `DCHECK` macros. Is there a purpose to removing it?